metadata/index.html

```diff
@@ -880,6 +852,10 @@
         <p>
           The description MAY contain <a title="inherited property">inherited properties</a> to describe cells within tables that use this schema.
         </p>
+        <section>
+          <h4>Schema Compatibility</h4>
+          <p>Two Schemas are <dfn title="schema compatible">compatible</dfn> if they have the same number of non-<a href="#column-virtual"><code>virtual</code></a> columns, and all non-<a href="#column-virtual"><code>virtual</code></a> columns are compatible with each other. Columns at the same index are compatible if there is a non-empty case-insensitive intersection between the percent-decoded <a href="#column-name"><code>name</code></a> and <a href="#column-titles"><code>titles</code></a> values.</p>
+        </section>
```

Comment:
That is, in fact, what we're doing. This allows metadata to define `name`, and compare with the equivalent `titles` from embedded metadata, for example. This text is lifted directly from the previous merge language on schema. The mechanism is to create an array containing the percent-decoded name, and all string values from `titles`, and see if there's an intersection.

Without comparing `name` against `titles`, every metadata column needs both `name` and `titles` annotations, which is overly redundant.